acme/setup.py

```diff
@@ -9,21 +9,18 @@
 
 # Please update tox.ini when modifying dependency version requirements
 install_requires = [
-    # load_pem_private/public_key (>=0.6)
-    # rsa_recover_prime_factors (>=0.8)
-    'cryptography>=1.2.3',
+    'cryptography>=2.1.4',
     # formerly known as acme.jose:
     # 1.1.0+ is required to avoid the warnings described at
     # https://github.com/certbot/josepy/issues/13.
     'josepy>=1.1.0',
-    # Connection.set_tlsext_host_name (>=0.13) + matching Xenial requirements (>=0.15.1)
-    'PyOpenSSL>=0.15.1',
+    'PyOpenSSL>=17.3.0',
     'pyrfc3339',
     'pytz',
     'requests[security]>=2.6.0',  # security extras added in 2.4.1
     'requests-toolbelt>=0.3.0',
-    'setuptools',
-    'six>=1.9.0',  # needed for python_2_unicode_compatible
+    'setuptools>=39.0.1',
+    'six>=1.11.0',
```

Comment:
Given the current code in `acme`, I assume you removed the comment because `six` is used for a lot more than `python_2_unicode_compatible` decorator, right?

Reply:
I assumed this comment was documenting why we need `six>=1.9.0` which looking at [`six`'s changelog](https://github.com/benjaminp/six/blob/ae22dcf6f3083cd02139145fe23f72cbf0d4913e/CHANGES#L88-L94) seems right. Since we're changing the minimum required version and why we're requiring that version is documented in `oldest_constraints.txt`, I figure it makes sense to remove it.